app/controllers/users_controller.rb

```diff
@@ -0,0 +1,26 @@
+class UsersController < ApplicationController
+  before_action :require_login, only: [:index, :destroy]
+
+  def index
+    @users = User.all.order(created_at: :desc)
+    @user = current_user
```

Comment:
You can just call `current_user` in the view so you don't have this dangling variable here which isn't necessary.